serverless.yaml

```diff
@@ -0,0 +1,46 @@
+service: cart-api-natallia-a
+
+plugins:
+  - serverless-offline
+
+useDotenv: true
+
+provider:
+  name: aws
+  stage: 'dev'
+  runtime: nodejs14.x
+  region: 'eu-central-1'
+  vpc:
+    securityGroupIds:
+      - 'sg-02046ca8c1f7f5d46'
+      - 'sg-02046ca8c1f7f5d46'
+    subnetIds:
+      - 'subnet-7d955331'
+      - 'subnet-0417926e'
+      - 'subnet-cb5aaab7'
+  environment:
+    PG_HOST: ${env:PG_HOST}
```

Comment:
As an alternative to `useDotenv: true`, you could use [serverless-dotenv-plugin](https://www.serverless.com/plugins/serverless-dotenv-plugin). 
It will automatically import all variables from `.env` into functions and there will be no need to pass them manually